src/config.ts

```diff
@@ -109,6 +110,10 @@ const config: Config = {
         message_id: "841608870001508372",
       },
     ],
+    excludedChannels: [
+      '748546478825275435',
+      '749651628067651674'
+    ]
```

Comment:
Could we save these strings as variables whose names line up with the channels?

That way, we could do:
```suggestion
    excludedChannels: [
      introductionsChannel,
      secretStaffChannel
    ]
```

I think this might make it easier to adjust down the road if need be.

Reply:
Seems like we both had the same thought. :D 

That would be fine for me as well, of course, I'll leave it up to you @SketchNI